src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -90,19 +87,20 @@ internal Process TryGetProcess()
                     lazyRemoteService.InitializedService.TryGetValue(out initializedService)) ? initializedService.ServiceOpt.Process : null;
         }
 
-        internal Service TryGetService()
+        /*internal Service TryGetService()
         {
             var initializedService = TryGetOrCreateRemoteServiceAsync().Result;
             return initializedService.ServiceOpt?.Service;
-        }
+        }*/
 
         // Triggered whenever we create a fresh process.
         // The ProcessExited event is not hooked yet.
         internal event Action<Process> InteractiveHostProcessCreated;
 
-        internal IpcServerChannel _ServerChannel
+        // (miziga) needed at all? instead of server channel return the stream?
+        internal NamedPipeClientStream _namedPipeClient
         {
-            get { return _serverChannel; }
+            get { return _clientStream; }
         }
```

Comment:
This shouldn't be necessary. I think some tests needed this before but shouldn't be needed now.